gptel-transient.el

```diff
@@ -397,20 +402,27 @@ Also format its value in the Transient menu."
        ;; and infixes when going through `gptel-menu'.
        ;; TODO: Raise an issue with Transient.
        collect (list (key-description (list key))
-                     (concat (capitalize name) " "
-                             (propertize " " 'display '(space :align-to 20))
-                             (propertize
-                              (concat
-                               "("
-                               (string-replace
-                                "\n" " "
-                                (truncate-string-to-width prompt (- width 30) nil nil t))
-                               ")")
-                              'face 'shadow))
+                     (if prompt
```

Comment:
I think it should be possible to disable the system message from the transient menu.  It should probably be a standalone option in the menu, bound to `<delete>` or something.